projects/admin/src/app/api/import-source-api.service.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 UCLouvain
+ * Copyright (C) 2021 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { ExternalSourceSetting } from '../classes/external-source';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ImportSourceApiService {
+
+  /**
+   * Constructor
+   * @param _httpClient - HttpClient
+   */
+  constructor(
+    private _httpClient: HttpClient
+  ) { }
+
+  /**
+   * Get the configuration entrypoint for external sources
+   * @return: an Observable on external sources list
+   */
+  getSources(): Observable<Array<ExternalSourceSetting>> {
```

Comment:
`Observable<ExternalSourceSetting[]>`

Reply:
@Garfield-fr Tsssss :D It's just the same write in a different way :D